scripts/checknodiffgenerated.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
```

Comment:
I know it's opinion-based, but we should use a convention in naming things - in this case, use underscores to separate words in the script name.

Reply:
The name was taken from Buf project and since we have bunch of scripts without any spaces/underscores I left it like this for now. Agreed on the consistency though and it would be good idea to clear all the names sometime.

Reply:
I see, ok